samples/compute/v2/servers/attach_detach_volume_attachment.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+use OpenStack\BlockStorage\v2\Models\VolumeAttachment;
```

Comment:
can we split this into 2 files, so: `attach_volume_attachment.php` and `detach_volume_attachment.php`